core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -571,7 +572,7 @@ public void remove() {
    */
   public static <T> void invokeAll(ExecutorService service, List<Callable<T>> callables,
       long timeoutMs) throws TimeoutException, ExecutionException {
-    long endMs = System.currentTimeMillis() + timeoutMs;
+    long endMs = getCurrentMs() + timeoutMs;
```

Comment:
`System.currentTimeMillis()` should all be changed to use the util method to hide implementation